diff -up qtwebengine-everywhere-src-5.15.10-20220523/src/3rdparty/chromium/third_party/skia/src/opts/SkRasterPipeline_opts.h.16~ qtwebengine-everywhere-src-5.15.10-20220523/src/3rdparty/chromium/third_party/skia/src/opts/SkRasterPipeline_opts.h
--- qtwebengine-everywhere-src-5.15.10-20220523/src/3rdparty/chromium/third_party/skia/src/opts/SkRasterPipeline_opts.h.16~	2022-05-23 23:04:18.806955225 +0200
+++ qtwebengine-everywhere-src-5.15.10-20220523/src/3rdparty/chromium/third_party/skia/src/opts/SkRasterPipeline_opts.h	2022-05-23 23:05:23.282971708 +0200
@@ -1004,7 +1004,7 @@ SI F from_half(U16 h) {
 }
 
 SI U16 to_half(F f) {
-#if defined(JUMPER_IS_NEON) && defined(SK_CPU_ARM64) \
+#if 0 && defined(JUMPER_IS_NEON) && defined(SK_CPU_ARM64) \
     && !defined(SK_BUILD_FOR_GOOGLE3)  // Temporary workaround for some Google3 builds.
 #if defined(SK_BUILD_FOR_MAC)
     return vcvt_f16_f32(f);
