diff -up qtwebengine-everywhere-src-5.15.0-20200311/src/3rdparty/chromium/base/task/promise/promise_executor.h.omv~ qtwebengine-everywhere-src-5.15.0-20200311/src/3rdparty/chromium/base/task/promise/promise_executor.h
--- qtwebengine-everywhere-src-5.15.0-20200311/src/3rdparty/chromium/base/task/promise/promise_executor.h.omv~	2020-03-11 04:19:02.606403985 +0100
+++ qtwebengine-everywhere-src-5.15.0-20200311/src/3rdparty/chromium/base/task/promise/promise_executor.h	2020-03-11 04:26:30.330382264 +0100
@@ -157,7 +157,7 @@ class BASE_EXPORT PromiseExecutor {
     void (*execute)(void* self, AbstractPromise* promise);
 
    private:
-    DISALLOW_COPY_AND_ASSIGN(VTable);
+//    DISALLOW_COPY_AND_ASSIGN(VTable);
   };
 
   template <typename DerivedType>
