diff -up qtwebengine-everywhere-src-5.15.1/src/core/web_engine_context.cpp.11~ qtwebengine-everywhere-src-5.15.1/src/core/web_engine_context.cpp
--- qtwebengine-everywhere-src-5.15.1/src/core/web_engine_context.cpp.11~	2020-09-01 13:02:34.000000000 +0200
+++ qtwebengine-everywhere-src-5.15.1/src/core/web_engine_context.cpp	2020-09-09 23:15:42.011145234 +0200
@@ -127,6 +127,8 @@
 #include <QOffscreenSurface>
 #if QT_CONFIG(opengl)
 # include <QOpenGLContext>
+# include <QOpenGLFunctions>
+# include <QOffscreenSurface>
 #endif
 #include <QQuickWindow>
 #include <QStringList>
@@ -186,6 +188,37 @@ void dummyGetPluginCallback(const std::v
 }
 #endif
 
+QString openGLVendor()
+{
+    QString vendor;
+
+    QOpenGLContext *oldContext = QOpenGLContext::currentContext();
+    QSurface *oldSurface = 0;
+    if (oldContext)
+        oldSurface = oldContext->surface();
+
+    QScopedPointer<QOffscreenSurface> surface( new QOffscreenSurface );
+    surface->create();
+    QOpenGLContext context;
+    if (!context.create()) {
+        qDebug() << "Error creating openGL context";
+    }
+    else if (!context.makeCurrent(surface.data())) {
+        qDebug() << "Error making openGL context current context";
+    } else {
+        const GLubyte *p;
+        QOpenGLFunctions *f = context.functions();
+        if ((p = f->glGetString(GL_VENDOR)))
+            vendor = QString::fromLatin1(reinterpret_cast<const char *>(p));
+    }
+
+    context.doneCurrent();
+    if (oldContext && oldSurface)
+        oldContext->makeCurrent(oldSurface);
+
+    return vendor;
+}
+
 } // namespace
 
 namespace QtWebEngineCore {
@@ -696,8 +729,17 @@ WebEngineContext::WebEngineContext()
 
     const char *glType = 0;
 #if QT_CONFIG(opengl)
-
-    const bool tryGL = (usingDefaultSGBackend() && !usingSoftwareDynamicGL() &&
+    bool disableGpu = qEnvironmentVariableIsSet("QT_WEBENGINE_DISABLE_GPU");
+    if (!qEnvironmentVariableIsSet("QT_WEBENGINE_DISABLE_NOUVEAU_WORKAROUND") && openGLVendor() == QStringLiteral("nouveau")) {
+	    qWarning() << "Nouveau openGL driver detected. Qt WebEngine will disable usage of the GPU.\n"
+		          "Please consider getting hardware not made by NVIDIA.\n\n"
+			  "Alternatively, you can set the QT_WEBENGINE_DISABLE_NOUVEAU_WORKAROUND\n"
+			  "environment variable before running this application, but this is \n"
+			  "not recommended since this usually causes applications to crash as\n"
+			  "Nouveau openGL drivers don't support multithreaded rendering";
+	    disableGpu = true;
+    }
+    const bool tryGL = (usingDefaultSGBackend() && !usingSoftwareDynamicGL() && !disableGpu &&
                         QGuiApplicationPrivate::platformIntegration()->hasCapability(QPlatformIntegration::OpenGL))
                         || enableGLSoftwareRendering;
     if (tryGL) {
