diff -up qtwebengine-everywhere-src-5.12.1/src/core/web_engine_context.cpp.5~ qtwebengine-everywhere-src-5.12.1/src/core/web_engine_context.cpp
--- qtwebengine-everywhere-src-5.12.1/src/core/web_engine_context.cpp.5~	2019-01-16 23:42:58.000000000 +0100
+++ qtwebengine-everywhere-src-5.12.1/src/core/web_engine_context.cpp	2019-02-06 12:45:52.942746708 +0100
@@ -102,6 +102,8 @@
 #include <QOffscreenSurface>
 #ifndef QT_NO_OPENGL
 # include <QOpenGLContext>
+# include <QOpenGLFunctions>
+# include <QOffscreenSurface>
 #endif
 #include <QQuickWindow>
 #include <QStringList>
@@ -161,6 +163,37 @@ void dummyGetPluginCallback(const std::v
 }
 #endif
 
+QString openGLVendor()
+{
+    QString vendor;
+
+    QOpenGLContext *oldContext = QOpenGLContext::currentContext();
+    QSurface *oldSurface = 0;
+    if (oldContext)
+        oldSurface = oldContext->surface();
+
+    QScopedPointer<QOffscreenSurface> surface( new QOffscreenSurface );
+    surface->create();
+    QOpenGLContext context;
+    if (!context.create()) {
+        qDebug() << "Error creating openGL context";
+    }
+    else if (!context.makeCurrent(surface.data())) {
+        qDebug() << "Error making openGL context current context";
+    } else {
+        const GLubyte *p;
+        QOpenGLFunctions *f = context.functions();
+        if ((p = f->glGetString(GL_VENDOR)))
+            vendor = QString::fromLatin1(reinterpret_cast<const char *>(p));
+    }
+
+    context.doneCurrent();
+    if (oldContext && oldSurface)
+        oldContext->makeCurrent(oldSurface);
+
+    return vendor;
+}
+
 } // namespace
 
 namespace QtWebEngineCore {
@@ -448,6 +481,16 @@ WebEngineContext::WebEngineContext()
 
     const char *glType = 0;
 #ifndef QT_NO_OPENGL
+    bool disableGpu = qEnvironmentVariableIsSet("QT_WEBENGINE_DISABLE_GPU");
+    if (!qEnvironmentVariableIsSet("QT_WEBENGINE_DISABLE_NOUVEAU_WORKAROUND") && openGLVendor() == QStringLiteral("nouveau")) {
+	    qWarning() << "Nouveau openGL driver detected. Qt WebEngine will disable usage of the GPU.\n"
+		          "Please consider getting hardware not made by NVIDIA.\n\n"
+			  "Alternatively, you can set the QT_WEBENGINE_DISABLE_NOUVEAU_WORKAROUND\n"
+			  "environment variable before running this application, but this is \n"
+			  "not recommended since this usually causes applications to crash as\n"
+			  "Nouveau openGL drivers don't support multithreaded rendering";
+	    disableGpu = true;
+    }
 
     bool tryGL =
             !usingANGLE()
@@ -457,7 +500,7 @@ WebEngineContext::WebEngineContext()
                 // performant, but at least provides WebGL support.
                 || enableWebGLSoftwareRendering
                 )
-            && !usingQtQuick2DRenderer();
+            && !usingQtQuick2DRenderer() && !disableGpu;
 
     if (tryGL) {
         if (qt_gl_global_share_context() && qt_gl_global_share_context()->isValid()) {
