diff -up qtwebengine-everywhere-src-5.15.0-20200311/src/3rdparty/chromium/media/base/decoder_buffer.h.15~ qtwebengine-everywhere-src-5.15.0-20200311/src/3rdparty/chromium/media/base/decoder_buffer.h
--- qtwebengine-everywhere-src-5.15.0-20200311/src/3rdparty/chromium/media/base/decoder_buffer.h.15~	2020-03-11 01:14:25.321941414 +0100
+++ qtwebengine-everywhere-src-5.15.0-20200311/src/3rdparty/chromium/media/base/decoder_buffer.h	2020-03-11 01:14:25.321941414 +0100
@@ -8,7 +8,9 @@
 #include <stddef.h>
 #include <stdint.h>
 
+extern "C" {
 #include <libavcodec/avcodec.h>
+}
 
 #include <memory>
 #include <string>
diff -up qtwebengine-everywhere-src-5.15.0-20200311/src/3rdparty/chromium/media/filters/BUILD.gn.15~ qtwebengine-everywhere-src-5.15.0-20200311/src/3rdparty/chromium/media/filters/BUILD.gn
--- qtwebengine-everywhere-src-5.15.0-20200311/src/3rdparty/chromium/media/filters/BUILD.gn.15~	2020-03-06 10:28:09.000000000 +0100
+++ qtwebengine-everywhere-src-5.15.0-20200311/src/3rdparty/chromium/media/filters/BUILD.gn	2020-03-11 01:16:58.966933960 +0100
@@ -104,6 +104,12 @@ jumbo_source_set("filters") {
     ]
   }
 
+  libs += [
+    "avcodec",
+    "avformat",
+    "avutil",
+  ]
+
   if (media_use_ffmpeg) {
     deps += [
       "//media/ffmpeg",
@@ -130,6 +136,11 @@ jumbo_source_set("filters") {
       "media_file_checker.cc",
       "media_file_checker.h",
     ]
+    libs += [
+      "avcodec",
+      "avformat",
+      "avutil",
+    ]
   }
 
   if (media_use_libvpx) {
@@ -170,6 +181,12 @@ jumbo_source_set("filters") {
           "ffmpeg_h265_to_annex_b_bitstream_converter.cc",
           "ffmpeg_h265_to_annex_b_bitstream_converter.h",
         ]
+
+        libs += [
+          "avcodec",
+          "avformat",
+          "avutil",
+        ]
       }
     }
 
@@ -178,6 +195,12 @@ jumbo_source_set("filters") {
         "ffmpeg_video_decoder.cc",
         "ffmpeg_video_decoder.h",
       ]
+
+      libs += [
+        "avcodec",
+        "avformat",
+        "avutil",
+      ]
     }
   }
 
