--- qtwebengine-opensource-src-5.6.0/src/core/config/desktop_linux.pri.0002~	2016-03-23 19:31:20.198091954 +0100
+++ qtwebengine-opensource-src-5.6.0/src/core/config/desktop_linux.pri	2016-03-23 19:44:19.303513385 +0100
@@ -5,6 +5,34 @@ include(linux.pri)
 GYP_CONFIG += \
     desktop_linux=1 \
     enable_plugins=1 \
+    use_pulseaudio=1 \
+    linux_link_libpci=1 \
+    linux_link_gsettings=1 \
+    linux_link_libspeechd=1 \
+    use_system_re2=1 \
+    use_system_yasm=1 \
+    use_system_opus=1 \
+    use_system_zlib=1 \
+    use_system_speex=1 \
+    use_system_expat=1 \
+    use_system_snappy=1 \
+    use_system_libpng=1 \
+    use_system_libxml=1 \
+    use_system_libjpeg=1 \
+    use_system_libwebp=1 \
+    use_system_libxslt=1 \
+    use_system_libsrtp=1 \
+    use_system_jsoncpp=1 \
+    use_system_libevent=1 \
+    use_system_protobuf=1 \
+    use_system_harfbuzz=1 \
+    use_system_xdg_utils=1 \
+    use_system_ffmpeg=0 \
+    build_ffmpegsumo=1 \
+    fastbuild=1 \
+    remove_webcore_debug_symbols=1 \
+    proprietary_codecs=1 \
+    ffmpeg_branding=Chrome
 
 linux-clang: GYP_CONFIG += clang=1 host_clang=1 clang_use_chrome_plugins=0 make_clang_dir=/usr
 else: GYP_CONFIG += clang=0 host_clang=0
