diff -up qtwebengine-everywhere-src-5.15.0-beta1/src/3rdparty/chromium/third_party/pdfium/core/fxcodec/progressivedecoder.cpp.omv~ qtwebengine-everywhere-src-5.15.0-beta1/src/3rdparty/chromium/third_party/pdfium/core/fxcodec/progressivedecoder.cpp
--- qtwebengine-everywhere-src-5.15.0-beta1/src/3rdparty/chromium/third_party/pdfium/core/fxcodec/progressivedecoder.cpp.omv~	2020-03-02 04:04:54.146354167 +0100
+++ qtwebengine-everywhere-src-5.15.0-beta1/src/3rdparty/chromium/third_party/pdfium/core/fxcodec/progressivedecoder.cpp	2020-03-02 04:05:32.185355845 +0100
@@ -740,7 +740,7 @@ bool ProgressiveDecoder::BmpDetectImageT
   }
 
   uint32_t pitch = 0;
-  uint32_t neededData = 0;
+  size_t neededData = 0;
   if (!CFX_DIBitmap::CalculatePitchAndSize(m_SrcWidth, m_SrcHeight, format,
                                            &pitch, &neededData)) {
     m_status = FXCODEC_STATUS_ERR_FORMAT;
