diff -urN qtwebengine-opensource-src-5.9.0-beta4/src/3rdparty/chromium/tools/gn/args.cc tools/gn/args.cc
--- qtwebengine-opensource-src-5.9.0-beta4/src/3rdparty/chromium/tools/gn/args.cc	2017-04-27 12:50:25.000000000 +0200
+++ tools/gn/args.cc	2017-05-16 21:04:09.000000000 +0200
@@ -266,6 +266,7 @@
   static const char kX86[] = "x86";
   static const char kX64[] = "x64";
   static const char kArm[] = "arm";
+  static const char kArm64[] = "arm64";
   static const char kMips[] = "mipsel";
   static const char kS390X[] = "s390x";
   static const char kPPC64[] = "ppc64";
@@ -280,11 +280,16 @@
     arch = kX64;
   else if (os_arch.substr(0, 3) == "arm")
     arch = kArm;
+  else if (os_arch == "aarch64")
+    arch = kArm64;
   else if (os_arch == "mips")
     arch = kMips;
   else if (os_arch == "s390x")
     arch = kS390X;
-  else if (os_arch == "mips")
+  else if (os_arch == "ppc64" || os_arch == "ppc64le")
+    // We handle the endianness inside //build/config/host_byteorder.gni.
+    // This allows us to use the same toolchain as ppc64 BE
+    // and specific flags are included using the host_byteorder logic.
     arch = kPPC64;
   else
     CHECK(false) << "OS architecture not handled. (" << os_arch << ")";
