diff -up qtwebengine-everywhere-src-5.15.8-20211228/src/3rdparty/chromium/sandbox/linux/services/credentials.cc.omv~ qtwebengine-everywhere-src-5.15.8-20211228/src/3rdparty/chromium/sandbox/linux/services/credentials.cc
--- qtwebengine-everywhere-src-5.15.8-20211228/src/3rdparty/chromium/sandbox/linux/services/credentials.cc.omv~	2021-12-28 01:57:14.968948484 +0100
+++ qtwebengine-everywhere-src-5.15.8-20211228/src/3rdparty/chromium/sandbox/linux/services/credentials.cc	2021-12-28 02:05:47.161681292 +0100
@@ -98,7 +98,8 @@ bool ChrootToSafeEmptyDir() {
   // attempt this optimization.
   clone_flags |= CLONE_VM | CLONE_VFORK | CLONE_SETTLS;
 
-  char tls_buf[PTHREAD_STACK_MIN] = {0};
+  char *tls_buf = new char[PTHREAD_STACK_MIN];
+  *tls_buf = 0;
   tls = tls_buf;
 #endif
 
@@ -109,6 +110,7 @@ bool ChrootToSafeEmptyDir() {
   int status = -1;
   PCHECK(HANDLE_EINTR(waitpid(pid, &status, 0)) == pid);
 
+  delete[] tls_buf;
   return WIFEXITED(status) && WEXITSTATUS(status) == kExitSuccess;
 }
 
