diff -up qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/chromium/build/config/chromecast/BUILD.gn.8~ qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/chromium/build/config/chromecast/BUILD.gn
--- qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/chromium/build/config/chromecast/BUILD.gn.8~	2020-01-29 16:18:33.000000000 +0100
+++ qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/chromium/build/config/chromecast/BUILD.gn	2020-01-30 21:45:07.520774394 +0100
@@ -14,15 +14,6 @@ config("static_config") {
       # http://b/26390825
       "-Wl,--exclude-libs=libffmpeg.a",
     ]
-
-    if (!is_android) {
-      ldflags += [
-        # We want to statically link libstdc++/libgcc on Linux.
-        # (On Android, libstdc++ and libgcc aren't used.)
-        "-static-libstdc++",
-        "-static-libgcc",
-      ]
-    }
   }
 }
 
diff -up qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/gn/build/gen.py.8~ qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/gn/build/gen.py
--- qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/gn/build/gen.py.8~	2020-01-30 21:45:07.521774394 +0100
+++ qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/gn/build/gen.py	2020-01-30 21:45:57.857776613 +0100
@@ -96,7 +96,7 @@ def main(argv):
   parser.add_option('--no-strip', action='store_true',
                     help='Don\'t strip release build. Useful for profiling.')
   parser.add_option('--no-static-libstdc++', action='store_true',
-                    default=False, dest='no_static_libstdcpp',
+                    default=True, dest='no_static_libstdcpp',
                     help='Don\'t link libstdc++ statically')
   parser.add_option('--cc',
                     help='The path to cc compiler.')
@@ -359,8 +359,8 @@ def WriteGNNinja(path, platform, host, o
     if platform.is_linux():
       ldflags.append('-Wl,--as-needed')
 
-      if not options.no_static_libstdcpp:
-        ldflags.append('-static-libstdc++')
+      # if not options.no_static_libstdcpp:
+      #  ldflags.append('-static-libstdc++')
 
       # This is needed by libc++.
       libs.append('-ldl')
diff -up qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/gn/infra/recipes/gn.expected/cipd_exists.json.8~ qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/gn/infra/recipes/gn.expected/cipd_exists.json
diff -up qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/gn/infra/recipes/gn.expected/cipd_register.json.8~ qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/gn/infra/recipes/gn.expected/cipd_register.json
