diff -up qtwebengine-everywhere-src-5.15.0-20200130/src/core/qtwebengine_sources.gni.omv~ qtwebengine-everywhere-src-5.15.0-20200130/src/core/qtwebengine_sources.gni
--- qtwebengine-everywhere-src-5.15.0-20200130/src/core/qtwebengine_sources.gni.omv~	2020-01-30 15:51:44.906839466 +0100
+++ qtwebengine-everywhere-src-5.15.0-20200130/src/core/qtwebengine_sources.gni	2020-01-30 15:51:55.163839918 +0100
@@ -128,8 +128,8 @@ source_set("qtwebengine_sources") {
 
   if (is_linux) {
     sources += [
-      "//chrome/browser/ui/webui/sandbox_internals_ui.cc",
-      "//chrome/browser/ui/webui/sandbox_internals_ui.h",
+      "//chrome/browser/ui/webui/sandbox/sandbox_internals_ui.cc",
+      "//chrome/browser/ui/webui/sandbox/sandbox_internals_ui.h",
     ]
   }
 
diff -up qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/chromium/third_party/blink/renderer/modules/peerconnection/peer_connection_dependency_factory.cc.omv~ qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/chromium/third_party/blink/renderer/modules/peerconnection/peer_connection_dependency_factory.cc
--- qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/chromium/third_party/blink/renderer/modules/peerconnection/peer_connection_dependency_factory.cc.omv~	2020-01-30 20:50:25.003629651 +0100
+++ qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/chromium/third_party/blink/renderer/modules/peerconnection/peer_connection_dependency_factory.cc	2020-01-30 20:50:43.669630474 +0100
@@ -167,7 +167,7 @@ void PeerConnectionDependencyFactory::Cr
 
 #if BUILDFLAG(RTC_USE_H264) && BUILDFLAG(ENABLE_FFMPEG_VIDEO_DECODERS)
   // Building /w |rtc_use_h264|, is the corresponding run-time feature enabled?
-  if (base::FeatureList::IsEnabled(kWebRtcH264WithOpenH264FFmpeg)) {
+  if (base::FeatureList::IsEnabled(features::kWebRtcH264WithOpenH264FFmpeg)) {
     // |H264DecoderImpl| may be used which depends on FFmpeg, therefore we need
     // to initialize FFmpeg before going further.
     media::FFmpegGlue::InitializeFFmpeg();
diff -up qtwebengine-everywhere-src-5.15.0-20200130/src/core/net/proxying_url_loader_factory_qt.cpp.omv~ qtwebengine-everywhere-src-5.15.0-20200130/src/core/net/proxying_url_loader_factory_qt.cpp
diff -up qtwebengine-everywhere-src-5.15.0-20200130/src/core/net/proxying_url_loader_factory_qt.h.omv~ qtwebengine-everywhere-src-5.15.0-20200130/src/core/net/proxying_url_loader_factory_qt.h
diff -up qtwebengine-everywhere-src-5.15.0-20200130/src/core/printing/print_view_manager_base_qt.cpp.omv~ qtwebengine-everywhere-src-5.15.0-20200130/src/core/printing/print_view_manager_base_qt.cpp
--- qtwebengine-everywhere-src-5.15.0-20200130/src/core/printing/print_view_manager_base_qt.cpp.omv~	2020-01-30 20:58:48.898651871 +0100
+++ qtwebengine-everywhere-src-5.15.0-20200130/src/core/printing/print_view_manager_base_qt.cpp	2020-01-30 20:59:03.052652495 +0100
@@ -150,7 +150,7 @@ printing::PrintedDocument *PrintViewMana
 
 // IPC handlers
 void PrintViewManagerBaseQt::OnDidPrintDocument(content::RenderFrameHost* /*render_frame_host*/,
-                                                const PrintHostMsg_DidPrintDocument_Params &params)
+                                                const PrintHostMsg_DidPrintDocument_Params &params, std::unique_ptr<DelayedFrameDispatchHelper> helper)
 {
     printing::PrintedDocument *document = GetDocument(params.document_cookie);
     if (!document)
diff -up qtwebengine-everywhere-src-5.15.0-20200130/src/core/printing/print_view_manager_base_qt.h.omv~ qtwebengine-everywhere-src-5.15.0-20200130/src/core/printing/print_view_manager_base_qt.h
--- qtwebengine-everywhere-src-5.15.0-20200130/src/core/printing/print_view_manager_base_qt.h.omv~	2020-01-30 20:58:24.834650809 +0100
+++ qtwebengine-everywhere-src-5.15.0-20200130/src/core/printing/print_view_manager_base_qt.h	2020-01-30 20:58:47.502651809 +0100
@@ -100,7 +100,8 @@ protected:
 
     // IPC Message handlers.
     void OnDidPrintDocument(content::RenderFrameHost* render_frame_host,
-                            const PrintHostMsg_DidPrintDocument_Params& params);
+                            const PrintHostMsg_DidPrintDocument_Params& params,
+			    std::unique_ptr<DelayedFrameDispatchHelper> helper) override;
     void OnShowInvalidPrinterSettingsError();
 
     // Processes a NOTIFY_PRINT_JOB_EVENT notification.
diff -up qtwebengine-everywhere-src-5.15.0-20200130/src/core/web_contents_view_qt.cpp.omv~ qtwebengine-everywhere-src-5.15.0-20200130/src/core/web_contents_view_qt.cpp
--- qtwebengine-everywhere-src-5.15.0-20200130/src/core/web_contents_view_qt.cpp.omv~	2020-01-30 21:02:22.847661305 +0100
+++ qtwebengine-everywhere-src-5.15.0-20200130/src/core/web_contents_view_qt.cpp	2020-01-30 21:02:36.674661914 +0100
@@ -111,7 +111,7 @@ content::RenderWidgetHostViewBase* WebCo
     return view;
 }
 
-void WebContentsViewQt::CreateView(const gfx::Size& initial_size, gfx::NativeView context)
+void WebContentsViewQt::CreateView(gfx::NativeView context)
 {
 }
 
diff -up qtwebengine-everywhere-src-5.15.0-20200130/src/core/web_contents_view_qt.h.omv~ qtwebengine-everywhere-src-5.15.0-20200130/src/core/web_contents_view_qt.h
--- qtwebengine-everywhere-src-5.15.0-20200130/src/core/web_contents_view_qt.h.omv~	2020-01-30 21:02:15.137660965 +0100
+++ qtwebengine-everywhere-src-5.15.0-20200130/src/core/web_contents_view_qt.h	2020-01-30 21:02:21.437661242 +0100
@@ -72,7 +72,7 @@ public:
     // content::WebContentsView overrides:
     content::RenderWidgetHostViewBase *CreateViewForWidget(content::RenderWidgetHost* render_widget_host, bool is_guest_view_hack) override;
 
-    void CreateView(const gfx::Size& initial_size, gfx::NativeView context) override;
+    void CreateView(gfx::NativeView context) override;
 
     content::RenderWidgetHostViewBase *CreateViewForChildWidget(content::RenderWidgetHost* render_widget_host) override;
 
