diff -up qtwebengine-everywhere-src-5.13.0/src/3rdparty/chromium/third_party/webrtc/rtc_base/physical_socket_server.cc.7~ qtwebengine-everywhere-src-5.13.0/src/3rdparty/chromium/third_party/webrtc/rtc_base/physical_socket_server.cc
--- qtwebengine-everywhere-src-5.13.0/src/3rdparty/chromium/third_party/webrtc/rtc_base/physical_socket_server.cc.7~	2019-05-23 14:39:34.000000000 +0200
+++ qtwebengine-everywhere-src-5.13.0/src/3rdparty/chromium/third_party/webrtc/rtc_base/physical_socket_server.cc	2019-07-04 16:40:22.791740926 +0200
@@ -28,6 +28,7 @@
 #include <sys/ioctl.h>
 #include <sys/select.h>
 #include <sys/time.h>
+#include <sys/socket.h>
 #include <unistd.h>
 #endif
 
@@ -68,6 +69,10 @@ typedef void* SockOptArg;
 
 #if defined(WEBRTC_POSIX) && !defined(WEBRTC_MAC) && !defined(__native_client__)
 
+#ifndef SIOCGSTAMP
+#define SIOCGSTAMP 0x8906
+#endif
+
 int64_t GetSocketRecvTimestamp(int socket) {
   struct timeval tv_ioctl;
   int ret = ioctl(socket, SIOCGSTAMP, &tv_ioctl);
