diff -up qtwebengine-everywhere-src-5.15.16-20231006/src/3rdparty/chromium/third_party/blink/renderer/platform/text/text_break_iterator.cc.omv~ qtwebengine-everywhere-src-5.15.16-20231006/src/3rdparty/chromium/third_party/blink/renderer/platform/text/text_break_iterator.cc
--- qtwebengine-everywhere-src-5.15.16-20231006/src/3rdparty/chromium/third_party/blink/renderer/platform/text/text_break_iterator.cc.omv~	2023-10-06 23:53:42.999237336 +0200
+++ qtwebengine-everywhere-src-5.15.16-20231006/src/3rdparty/chromium/third_party/blink/renderer/platform/text/text_break_iterator.cc	2023-10-06 23:54:13.134526560 +0200
@@ -216,6 +216,11 @@ static const unsigned char kBreakAllLine
     { B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0) }, // CJ
     { B(0, 1, 1, 0, 1, 0, 0, 0), B(0, 0, 0, 0, 0, 1, 0, 0), B(0, 0, 0, 1, 1, 0, 1, 0), B(1, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 1, 0) }, // HL
     { B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0) }, // RI
+    { B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0) }, // Don't break the assert
+    { B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0) }, // Don't break the assert
+    { B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0) }, // Don't break the assert
+    { B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0) }, // Don't break the assert
+    { B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0), B(0, 0, 0, 0, 0, 0, 0, 0) }, // Don't break the assert
 };
 // clang-format on
 
diff -up qtwebengine-everywhere-src-5.15.16-20231006/src/3rdparty/chromium/third_party/icu/BUILD.gn.omv~ qtwebengine-everywhere-src-5.15.16-20231006/src/3rdparty/chromium/third_party/icu/BUILD.gn
--- qtwebengine-everywhere-src-5.15.16-20231006/src/3rdparty/chromium/third_party/icu/BUILD.gn.omv~	2023-10-07 00:49:14.039919069 +0200
+++ qtwebengine-everywhere-src-5.15.16-20231006/src/3rdparty/chromium/third_party/icu/BUILD.gn	2023-10-07 00:51:42.520380621 +0200
@@ -10,7 +10,7 @@ import("//third_party/icu/config.gni")
 import("//third_party/icu/sources.gni")
 
 declare_args() {
-  use_system_icu = false
+  use_system_icu = true
 }
 
 if (is_android) {
