diff -up qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/media/gpu/vaapi/vaapi_picture_tfp.cc.omv~ qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/media/gpu/vaapi/vaapi_picture_tfp.cc
--- qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/media/gpu/vaapi/vaapi_picture_tfp.cc.omv~	2020-04-23 16:51:08.620821273 +0200
+++ qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/media/gpu/vaapi/vaapi_picture_tfp.cc	2020-04-23 16:51:32.015623924 +0200
@@ -55,7 +55,7 @@ bool VaapiTFPPicture::Initialize() {
   if (make_context_current_cb_ && !make_context_current_cb_.Run())
     return false;
 
-  glx_image_ = new gl::GLImageGLX(size_, GL_RGB);
+  glx_image_ = new gl::GLImageGLX(size_, glx::BufferFormat::BGRX_8888);
   if (!glx_image_->Initialize(x_pixmap_)) {
     // x_pixmap_ will be freed in the destructor.
     DLOG(ERROR) << "Failed creating a GLX Pixmap for TFP";
diff -up qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/media/mojo/services/gpu_mojo_media_client.cc.omv~ qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/media/mojo/services/gpu_mojo_media_client.cc
--- qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/media/mojo/services/gpu_mojo_media_client.cc.omv~	2020-04-23 16:50:11.380294867 +0200
+++ qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/media/mojo/services/gpu_mojo_media_client.cc	2020-04-23 16:50:54.559938859 +0200
@@ -160,7 +160,7 @@ GpuMojoMediaClient::GetSupportedVideoDec
   supported_config_map[VideoDecoderImplementation::kAlternate] =
       *d3d11_supported_configs_;
 
-#elif defined(USE_CHROMEOS_MEDIA_ACCELERATION)
+#elif defined(OS_CHROMEOS)
   if (base::FeatureList::IsEnabled(kChromeosVideoDecoder)) {
     if (!cros_supported_configs_) {
       cros_supported_configs_ =
diff -up qtwebengine-everywhere-src-5.15.0-beta4/src/buildtools/config/linux.pri.omv~ qtwebengine-everywhere-src-5.15.0-beta4/src/buildtools/config/linux.pri
--- qtwebengine-everywhere-src-5.15.0-beta4/src/buildtools/config/linux.pri.omv~	2020-04-23 16:46:48.402845903 +0200
+++ qtwebengine-everywhere-src-5.15.0-beta4/src/buildtools/config/linux.pri	2020-04-23 16:49:43.619519468 +0200
@@ -216,6 +216,8 @@ host_build {
     qtConfig(webengine-system-re2):      gn_args += use_system_re2=true
     qtConfig(webengine-system-lcms2):    gn_args += use_system_lcms2=true
 
+    qtConfig(webengine-vaapi):           gn_args += use_vaapi=true
+
     # FIXME:
     #qtConfig(webengine-system-protobuf): gn_args += use_system_protobuf=true
     #qtConfig(webengine-system-jsoncpp): gn_args += use_system_jsoncpp=true
diff -up qtwebengine-everywhere-src-5.15.0-beta4/src/buildtools/configure.json.omv~ qtwebengine-everywhere-src-5.15.0-beta4/src/buildtools/configure.json
--- qtwebengine-everywhere-src-5.15.0-beta4/src/buildtools/configure.json.omv~	2020-04-23 17:30:20.145966423 +0200
+++ qtwebengine-everywhere-src-5.15.0-beta4/src/buildtools/configure.json	2020-04-23 17:32:23.901675124 +0200
@@ -661,7 +661,12 @@
                 "privateFeature",
                 { "type": "varAssign", "name": "PLATFORM_ERROR", "value": "tests.webengine-platform-support.platformSupport" }
             ]
-        }
+        },
+        "webengine-vaapi": {
+            "label": "vaapi",
+            "condition": "config.linux",
+            "output": [ "privateFeature" ]
+	}
     },
     "report": [
         {
diff -up qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/ui/gfx/BUILD.gn.omv~ qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/ui/gfx/BUILD.gn
--- qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/ui/gfx/BUILD.gn.omv~	2020-04-23 21:23:37.525189987 +0200
+++ qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/ui/gfx/BUILD.gn	2020-04-23 21:23:53.901560545 +0200
@@ -584,7 +584,7 @@ jumbo_source_set("memory_buffer_sources"
     "//ui/gfx/geometry",
   ]
 
-  if (is_linux && !use_qt) {
+  if (is_linux) {
     sources += [
       "linux/client_native_pixmap_dmabuf.cc",
       "linux/client_native_pixmap_dmabuf.h",
diff -up qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/media/gpu/vaapi/BUILD.gn.omv~ qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/media/gpu/vaapi/BUILD.gn
--- qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/media/gpu/vaapi/BUILD.gn.omv~	2020-04-24 10:49:46.857188706 +0200
+++ qtwebengine-everywhere-src-5.15.0-beta4/src/3rdparty/chromium/media/gpu/vaapi/BUILD.gn	2020-04-24 10:50:13.716288097 +0200
@@ -134,7 +134,7 @@ source_set("vaapi") {
       "vaapi_picture_native_pixmap.cc",
       "vaapi_picture_native_pixmap.h",
     ]
-    if (ozone_platform_gbm) {
+    if (ozone_platform_gbm || use_vaapi) {
       sources += [
         "vaapi_picture_native_pixmap_ozone.cc",
         "vaapi_picture_native_pixmap_ozone.h",
