diff -up qtwebengine-everywhere-src-5.12.1/mkspecs/features/functions.prf.2~ qtwebengine-everywhere-src-5.12.1/mkspecs/features/functions.prf
--- qtwebengine-everywhere-src-5.12.1/mkspecs/features/functions.prf.2~	2019-02-06 12:40:23.693716277 +0100
+++ qtwebengine-everywhere-src-5.12.1/mkspecs/features/functions.prf	2019-02-06 12:42:37.285854197 +0100
@@ -11,7 +11,7 @@ defineReplace(getChromiumSrcDir) {
 }
 
 defineReplace(extractCFlag) {
-    CFLAGS = $$QMAKE_CC $$QMAKE_CFLAGS
+    CFLAGS = $$QMAKE_CC $$QMAKE_CFLAGS $$QMAKE_CFLAGS_RELEASE
     OPTION = $$find(CFLAGS, $$1)
     OPTION = $$split(OPTION, =)
     PARAM = $$member(OPTION, 1)
