--- qtwebengine-opensource-src-5.9.0-beta/src/core/config/linux.pri.0001~	2017-04-09 20:47:09.587401731 +0200
+++ qtwebengine-opensource-src-5.9.0-beta/src/core/config/linux.pri	2017-04-09 20:48:21.377930931 +0200
@@ -148,3 +148,14 @@ host_build {
     use?(system_ffmpeg):   gn_args += use_system_ffmpeg=true
     #use?(system_protobuf): gn_args += use_system_protobuf=true
 }
+
+# link libpci instead of dlopening it, our Qt packaging depends on it anyway
+#gn_args += linux_link_libpci=true
+
+# run the unbundling script Chromium provides
+CHROMIUM_SRC_DIR = "$$QTWEBENGINE_ROOT/$$getChromiumSrcDir()"
+R_G_F_PY = "$$CHROMIUM_SRC_DIR/build/linux/unbundle/replace_gn_files.py"
+R_G_F_PY_ARGS = "--system-libraries re2 yasm"
+#for (config, GYP_CONFIG): R_G_F_PY_ARGS += "-D $$config"
+log("Running python $$R_G_F_PY $$R_G_F_PY_ARGS$${EOL}")
+!system("python $$R_G_F_PY $$R_G_F_PY_ARGS"): error("-- unbundling failed")
