diff -up qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/gn/build/gen.py.7~ qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/gn/build/gen.py
--- qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/gn/build/gen.py.7~	2020-01-30 15:45:15.601822300 +0100
+++ qtwebengine-everywhere-src-5.15.0-20200130/src/3rdparty/gn/build/gen.py	2020-01-30 15:45:40.282823388 +0100
@@ -96,7 +96,7 @@ def main(argv):
   parser.add_option('--no-strip', action='store_true',
                     help='Don\'t strip release build. Useful for profiling.')
   parser.add_option('--no-static-libstdc++', action='store_true',
-                    default=False, dest='no_static_libstdcpp',
+                    default=True, dest='no_static_libstdcpp',
                     help='Don\'t link libstdc++ statically')
   parser.add_option('--cc',
                     help='The path to cc compiler.')
