diff -up qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/baseline_policy.cc.18~ qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/baseline_policy.cc
--- qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/baseline_policy.cc.18~	2021-03-09 21:38:18.157356087 +0100
+++ qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/baseline_policy.cc	2021-03-09 21:38:18.157356087 +0100
@@ -157,7 +157,14 @@ ResultExpr EvaluateSyscallImpl(int fs_de
     return Allow();
 #endif
 
-  if (sysno == __NR_clock_gettime || sysno == __NR_clock_nanosleep) {
+  if (sysno == __NR_clock_gettime || sysno == __NR_clock_nanosleep
+#if defined(__NR_clock_gettime64)
+                                  || sysno == __NR_clock_gettime64
+#endif
+#if defined(__NR_clock_nanosleep_time64)
+                                  || sysno == __NR_clock_nanosleep_time64
+#endif
+  ) {
     return RestrictClockID();
   }
 
diff -up qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/syscall_parameters_restrictions_unittests.cc.18~ qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/syscall_parameters_restrictions_unittests.cc
--- qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/syscall_parameters_restrictions_unittests.cc.18~	2021-03-07 00:06:05.000000000 +0100
+++ qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/syscall_parameters_restrictions_unittests.cc	2021-03-09 21:38:18.157356087 +0100
@@ -60,6 +60,12 @@ class RestrictClockIdPolicy : public bpf
       case __NR_clock_gettime:
       case __NR_clock_getres:
       case __NR_clock_nanosleep:
+#if defined(__NR_clock_nanosleep_time64)
+      case __NR_clock_nanosleep_time64:
+#endif
+#if defined(__NR_clock_gettime64)
+      case __NR_clock_gettime64:
+#endif
         return RestrictClockID();
       default:
         return Allow();
diff -up qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/syscall_sets.cc.18~ qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/syscall_sets.cc
--- qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/syscall_sets.cc.18~	2021-03-07 00:06:05.000000000 +0100
+++ qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/syscall_sets.cc	2021-03-09 21:38:18.157356087 +0100
@@ -39,6 +39,12 @@ bool SyscallSets::IsAllowedGettime(int s
                                 // filtered by RestrictClokID().
     case __NR_clock_gettime:    // Parameters filtered by RestrictClockID().
     case __NR_clock_nanosleep:  // Parameters filtered by RestrictClockID().
+#if defined(__NR_clock_gettime64)
+    case __NR_clock_gettime64:  // Parameters filtered by RestrictClockID().
+#endif
+#if defined(__NR_clock_nanosleep_time64)
+    case __NR_clock_nanosleep_time64:  // Parameters filtered by RestrictClockID().
+#endif
     case __NR_clock_settime:    // Privileged.
 #if defined(__i386__) || \
     (defined(ARCH_CPU_MIPS_FAMILY) && defined(ARCH_CPU_32_BITS))
diff -up qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/system_headers/arm_linux_syscalls.h.18~ qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/system_headers/arm_linux_syscalls.h
--- qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/system_headers/arm_linux_syscalls.h.18~	2021-03-09 21:38:18.160689447 +0100
+++ qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/system_headers/arm_linux_syscalls.h	2021-03-09 21:38:52.554300342 +0100
@@ -1385,6 +1385,14 @@
 #define __NR_memfd_create (__NR_SYSCALL_BASE+385)
 #endif
 
+#if !defined(__NR_clock_gettime64)
+#define __NR_clock_gettime64 (__NR_SYSCALL_BASE+403)
+#endif
+
+#if !defined(__NR_clock_nanosleep_time64)
+#define __NR_clock_nanosleep_time64 (__NR_SYSCALL_BASE+407)
+#endif
+
 #if !defined(__NR_bpf)
 #define __NR_bpf (__NR_SYSCALL_BASE+386)
 #endif
diff -up qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/system_headers/mips_linux_syscalls.h.18~ qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/system_headers/mips_linux_syscalls.h
--- qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/system_headers/mips_linux_syscalls.h.18~	2021-03-07 00:06:05.000000000 +0100
+++ qtwebengine-everywhere-src-5.15.4-20210309/src/3rdparty/chromium/sandbox/linux/system_headers/mips_linux_syscalls.h	2021-03-09 21:38:18.160689447 +0100
@@ -1433,4 +1433,12 @@
 #define __NR_memfd_create (__NR_Linux + 354)
 #endif
 
+#if !defined(__NR_clock_gettime64)
+#define __NR_clock_gettime64 (__NR_Linux + 403)
+#endif
+
+#if !defined(__NR_clock_nanosleep_time64)
+#define __NR_clock_nanosleep_time64 (__NR_Linux + 407)
+#endif
+
 #endif  // SANDBOX_LINUX_SYSTEM_HEADERS_MIPS_LINUX_SYSCALLS_H_
