diff -up qtwebengine-everywhere-src-5.15.0-beta1/configure.pri.omv~ qtwebengine-everywhere-src-5.15.0-beta1/configure.pri
--- qtwebengine-everywhere-src-5.15.0-beta1/configure.pri.omv~	2020-03-01 16:28:44.663512345 +0100
+++ qtwebengine-everywhere-src-5.15.0-beta1/configure.pri	2020-03-01 16:29:05.585513268 +0100
@@ -166,7 +166,7 @@ defineTest(qtConfTest_detectNinja) {
     !isEmpty(ninja) {
         qtLog("Found ninja from path: $$ninja")
         qtRunLoggedCommand("$$ninja --version", version)|return(false)
-        contains(version, "1.[7-9].*"): return(true)
+        contains(version, "1.([7-9]|[0-9]+).*"): return(true)
         qtLog("Ninja version too old")
     }
     qtLog("Building own ninja")
