diff -up qtwebengine-everywhere-src-5.15.0-20200125/src/3rdparty/chromium/build/config/compiler/BUILD.gn.12~ qtwebengine-everywhere-src-5.15.0-20200125/src/3rdparty/chromium/build/config/compiler/BUILD.gn
--- qtwebengine-everywhere-src-5.15.0-20200125/src/3rdparty/chromium/build/config/compiler/BUILD.gn.12~	2020-01-25 17:29:11.199976722 +0100
+++ qtwebengine-everywhere-src-5.15.0-20200125/src/3rdparty/chromium/build/config/compiler/BUILD.gn	2020-01-25 17:30:05.177977270 +0100
@@ -2165,28 +2165,6 @@ config("default_optimization") {
 }
 
 _clang_sample_profile = ""
-if (is_clang && is_a_target_toolchain) {
-  if (clang_sample_profile_path != "") {
-    _clang_sample_profile = clang_sample_profile_path
-  } else if (clang_use_default_sample_profile) {
-    assert(build_with_chromium,
-           "Our default profiles currently only apply to Chromium")
-    assert(is_android || is_desktop_linux || is_chromeos,
-           "The current platform has no default profile")
-    if (is_android || is_desktop_linux) {
-      _clang_sample_profile = "//chrome/android/profiles/afdo.prof"
-    } else {
-      assert(
-          chromeos_afdo_platform == "silvermont" ||
-              chromeos_afdo_platform == "airmont" ||
-              chromeos_afdo_platform == "broadwell",
-          "Only silvermont, airmont and broadwell are valid Chrome OS profiles.")
-      _clang_sample_profile =
-          "//chromeos/profiles/${chromeos_afdo_platform}.afdo.prof"
-    }
-  }
-}
-
 # Clang offers a way to assert that AFDO profiles are accurate, which causes it
 # to optimize functions not represented in a profile more aggressively for size.
 # This config can be toggled in cases where shaving off binary size hurts
