diff -up qtwebengine-everywhere-src-5.12.0/src/3rdparty/chromium/build/config/compiler/BUILD.gn.7~ qtwebengine-everywhere-src-5.12.0/src/3rdparty/chromium/build/config/compiler/BUILD.gn
--- qtwebengine-everywhere-src-5.12.0/src/3rdparty/chromium/build/config/compiler/BUILD.gn.7~	2018-12-17 03:27:57.625953208 +0100
+++ qtwebengine-everywhere-src-5.12.0/src/3rdparty/chromium/build/config/compiler/BUILD.gn	2018-12-17 03:39:04.999789661 +0100
@@ -1432,6 +1432,7 @@ config("default_warnings") {
       # of Chromium and GCC bugs.
       cflags += [ "-Wno-dangling-else" ]
       cflags += [ "-Wno-packed-not-aligned" ]
+      cflags += [ "-Wno-class-memaccess" ]
       cflags_cc += [ "-Wno-attributes" ]
       cflags_cc += [ "-Wno-class-memaccess" ]
       cflags_cc += [ "-Wno-subobject-linkage" ]
