diff -up qtwebengine-everywhere-src-5.13.0-beta3/src/core/content_main_delegate_qt.cpp.12~ qtwebengine-everywhere-src-5.13.0-beta3/src/core/content_main_delegate_qt.cpp
--- qtwebengine-everywhere-src-5.13.0-beta3/src/core/content_main_delegate_qt.cpp.12~	2019-04-13 12:00:18.000000000 +0200
+++ qtwebengine-everywhere-src-5.13.0-beta3/src/core/content_main_delegate_qt.cpp	2019-05-07 16:57:24.879822083 +0200
@@ -226,9 +226,7 @@ static void SafeOverridePathImpl(const c
 bool ContentMainDelegateQt::BasicStartupComplete(int *exit_code)
 {
     SafeOverridePath(base::FILE_EXE, WebEngineLibraryInfo::getPath(base::FILE_EXE));
-#if ICU_UTIL_DATA_IMPL == ICU_UTIL_DATA_FILE
     SafeOverridePath(base::DIR_QT_LIBRARY_DATA, WebEngineLibraryInfo::getPath(base::DIR_QT_LIBRARY_DATA));
-#endif
     SafeOverridePath(ui::DIR_LOCALES, WebEngineLibraryInfo::getPath(ui::DIR_LOCALES));
 #if QT_CONFIG(webengine_spellchecker)
     SafeOverridePath(base::DIR_APP_DICTIONARIES, WebEngineLibraryInfo::getPath(base::DIR_APP_DICTIONARIES));
diff -up qtwebengine-everywhere-src-5.13.0-beta3/src/core/web_engine_library_info.cpp.12~ qtwebengine-everywhere-src-5.13.0-beta3/src/core/web_engine_library_info.cpp
--- qtwebengine-everywhere-src-5.13.0-beta3/src/core/web_engine_library_info.cpp.12~	2019-05-07 16:57:24.860822230 +0200
+++ qtwebengine-everywhere-src-5.13.0-beta3/src/core/web_engine_library_info.cpp	2019-05-07 16:58:02.444531055 +0200
@@ -257,18 +257,6 @@ QString dictionariesPath()
 }
 #endif // QT_CONFIG(webengine_spellchecker)
 
-QString icuDataPath()
-{
-    static QString potentialResourcesPath =
-#if defined(OS_MACOSX) && defined(QT_MAC_FRAMEWORK_BUILD)
-            getResourcesPath(frameworkBundle());
-#else
-            QLibraryInfo::location(QLibraryInfo::DataPath) % QLatin1String("/resources");
-#endif
-
-    return potentialResourcesPath;
-}
-
 QString resourcesDataPath()
 {
     static bool initialized = false;
@@ -325,7 +313,7 @@ base::FilePath WebEngineLibraryInfo::get
         directory = QStandardPaths::writableLocation(QStandardPaths::DesktopLocation);
         break;
     case base::DIR_QT_LIBRARY_DATA:
-        return toFilePath(icuDataPath());
+        return toFilePath(resourcesDataPath());
     case ui::DIR_LOCALES:
         return toFilePath(localesPath());
 #if QT_CONFIG(webengine_spellchecker)
