diff -up qtwebengine-everywhere-src-5.11.0-rc2/src/3rdparty/chromium/mojo/public/cpp/bindings/associated_interface_ptr_info.h.omv~ qtwebengine-everywhere-src-5.11.0-rc2/src/3rdparty/chromium/mojo/public/cpp/bindings/associated_interface_ptr_info.h
--- qtwebengine-everywhere-src-5.11.0-rc2/src/3rdparty/chromium/mojo/public/cpp/bindings/associated_interface_ptr_info.h.omv~	2018-05-21 16:08:11.354846206 +0200
+++ qtwebengine-everywhere-src-5.11.0-rc2/src/3rdparty/chromium/mojo/public/cpp/bindings/associated_interface_ptr_info.h	2018-05-21 16:09:07.133573651 +0200
@@ -45,7 +45,7 @@ class AssociatedInterfacePtrInfo {
 
   bool is_valid() const { return handle_.is_valid(); }
 
-  explicit operator bool() const { return handle_; }
+  explicit operator bool() const { return static_cast<bool>(handle_); }
 
   ScopedInterfaceEndpointHandle PassHandle() {
     return std::move(handle_);
