diff -up qtwebengine-everywhere-src-5.15.9-20220314/src/3rdparty/chromium/third_party/skia/src/utils/SkParseColor.cpp.omv~ qtwebengine-everywhere-src-5.15.9-20220314/src/3rdparty/chromium/third_party/skia/src/utils/SkParseColor.cpp
--- qtwebengine-everywhere-src-5.15.9-20220314/src/3rdparty/chromium/third_party/skia/src/utils/SkParseColor.cpp.omv~	2022-05-23 22:58:13.413825294 +0200
+++ qtwebengine-everywhere-src-5.15.9-20220314/src/3rdparty/chromium/third_party/skia/src/utils/SkParseColor.cpp	2022-05-23 22:58:34.574835026 +0200
@@ -7,6 +7,12 @@
 
 
 #include "include/utils/SkParse.h"
+#include "include/core/SkColor.h"
+#include "include/core/SkTypes.h"
+
+#include <string.h>
+#include <algorithm>
+#include <iterator>
 
 static constexpr const char* gColorNames[] = {
     "aliceblue",
