diff -up qtwebengine-everywhere-src-5.15.4-20210513/src/3rdparty/chromium/third_party/abseil-cpp/absl/synchronization/internal/graphcycles.cc.omv~ qtwebengine-everywhere-src-5.15.4-20210513/src/3rdparty/chromium/third_party/abseil-cpp/absl/synchronization/internal/graphcycles.cc
--- qtwebengine-everywhere-src-5.15.4-20210513/src/3rdparty/chromium/third_party/abseil-cpp/absl/synchronization/internal/graphcycles.cc.omv~	2021-05-13 19:47:34.521715059 +0200
+++ qtwebengine-everywhere-src-5.15.4-20210513/src/3rdparty/chromium/third_party/abseil-cpp/absl/synchronization/internal/graphcycles.cc	2021-05-13 19:47:40.868765199 +0200
@@ -37,6 +37,7 @@
 
 #include <algorithm>
 #include <array>
+#include <limits>
 #include "absl/base/internal/hide_ptr.h"
 #include "absl/base/internal/raw_logging.h"
 #include "absl/base/internal/spinlock.h"
diff -up qtwebengine-everywhere-src-5.15.4-20210513/src/3rdparty/chromium/third_party/perfetto/src/trace_processor/containers/string_pool.h.omv~ qtwebengine-everywhere-src-5.15.4-20210513/src/3rdparty/chromium/third_party/perfetto/src/trace_processor/containers/string_pool.h
--- qtwebengine-everywhere-src-5.15.4-20210513/src/3rdparty/chromium/third_party/perfetto/src/trace_processor/containers/string_pool.h.omv~	2021-05-13 19:51:26.378583230 +0200
+++ qtwebengine-everywhere-src-5.15.4-20210513/src/3rdparty/chromium/third_party/perfetto/src/trace_processor/containers/string_pool.h	2021-05-13 19:51:52.313796173 +0200
@@ -22,6 +22,8 @@
 
 #include <unordered_map>
 #include <vector>
+#include <limits>
+#include <cstdint>
 
 #include "perfetto/ext/base/optional.h"
 #include "perfetto/ext/base/paged_memory.h"
diff -up qtwebengine-everywhere-src-5.15.4-20210513/src/3rdparty/chromium/third_party/zlib/google/zip_internal.cc.omv~ qtwebengine-everywhere-src-5.15.4-20210513/src/3rdparty/chromium/third_party/zlib/google/zip_internal.cc
--- qtwebengine-everywhere-src-5.15.4-20210513/src/3rdparty/chromium/third_party/zlib/google/zip_internal.cc.omv~	2021-05-13 20:39:03.287496737 +0200
+++ qtwebengine-everywhere-src-5.15.4-20210513/src/3rdparty/chromium/third_party/zlib/google/zip_internal.cc	2021-05-13 20:39:39.165819059 +0200
@@ -13,10 +13,12 @@
 #include "base/notreached.h"
 #include "base/strings/utf_string_conversions.h"
 
-#if defined(USE_SYSTEM_MINIZIP)
+#if 1 //defined(USE_SYSTEM_MINIZIP)
+#include <zlib.h>
 #include <minizip/ioapi.h>
 #include <minizip/unzip.h>
 #include <minizip/zip.h>
+typedef unsigned long uLong;
 #else
 #include "third_party/zlib/contrib/minizip/unzip.h"
 #include "third_party/zlib/contrib/minizip/zip.h"
