diff -up qtwebengine-everywhere-src-5.12.0/src/3rdparty/chromium/third_party/skia/src/opts/SkRasterPipeline_opts.h.omv~ qtwebengine-everywhere-src-5.12.0/src/3rdparty/chromium/third_party/skia/src/opts/SkRasterPipeline_opts.h
--- qtwebengine-everywhere-src-5.12.0/src/3rdparty/chromium/third_party/skia/src/opts/SkRasterPipeline_opts.h.omv~	2018-12-22 03:06:53.442685008 +0000
+++ qtwebengine-everywhere-src-5.12.0/src/3rdparty/chromium/third_party/skia/src/opts/SkRasterPipeline_opts.h	2018-12-22 03:07:45.016840736 +0000
@@ -679,7 +679,7 @@ SI F from_half(U16 h) {
 }
 
 SI U16 to_half(F f) {
-#if defined(__ARM_FP16_FORMAT_IEEE)
+#if 0 && defined(__ARM_FP16_FORMAT_IEEE)
     __fp16 fp16 = __fp16(f);
     U16 u16;
     memcpy(&u16, &fp16, sizeof(U16));
